features/pdf-viewer.yml

```diff
@@ -0,0 +1,9 @@
+name: pdfViewerEnabled
+description: "The `navigator.pdfViewerEnabled` read-only property indicates whether the browser supports inline display of PDF files when navigating to them. If inline viewing is not supported the PDF is downloaded and may then be handled by an external application."
+spec: https://html.spec.whatwg.org/multipage/system-state.html#pdf-viewing-support
+# Caniuse has a feature for PDF viewing generally, which is marked not supported for
+# Chrome/Firefox Android, but also has this entry from the BCD key shown all green.
+# Not blocking on finding a resolution here, but should figure it out at some point.
+#caniuse: pdf-viewer
```

Comment:
I think the caniuse feature is about something else — whether the browser _can_ show PDFs inline, not whether its turned on and available. Can you remove all this and paste it an issue about it? I don't think it's really in-scope for us and would like to have that discussion elsewhere.

Reply:
#2069